### DISCUSSION TITLE
Can someone please help me.

### DISCUSSION THREAD

**Discussion by TheRenegade47** [Squidpad/BG3DevouringAndDigesting#8] [Q&A]
I am incredibly horrible with code but really want to try this, so if anyone would be willing to help me through the process, as I still have no idea what I'm doing, that would be absolutely wonderful. Preemptive thanks to anyone for considering helping.

**Comment by smeargle99** [comment]
Are you still trying to get the mod to work? What's been going wrong for you? I can try to walk you through it
